src/index.ts

```diff
@@ -53,10 +53,11 @@ const sidecarPlugin: JupyterFrontEndPlugin<void> = {
             });
           } else {
             const anchor = this.model.get('anchor') || 'right';
+            const ref = this.model.get('ref') || null;
```

Comment:
This is new to me, so would appreciate an explanation at a "for dummies" level :) How does `ref` get set? I see it being grabbed and passed, but I don't see it being set to anything other than `null`.